grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -457,6 +458,26 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
                   this.httpApiService.addClientLabel(clientId, label)),
           tap(() => this.fetchClient()),
           ));
+
+  /** An effect to remove a label from the selected client */
+  readonly removeClientLabel = this.effect<string>(
+      obs$ => obs$.pipe(
+          withLatestFrom(this.selectedClientId$),
+          switchMap(
+              ([label, clientId]) =>
+                  this.httpApiService.removeClientLabel(clientId, label)),
+          tap((label) => {
+            this.fetchClient();
+            this.removedClientLabels$.next(label);
```

Comment:
Please loop this through the store's state: Extend `ClientPageState` with a field (e.g. lastRemovedClientLabel`), which is then updated on successful removal (in the `tap`). You can ignore the errors for now - remove `catchError`.

removedClientLabels$ should not be a `new Subject`, instead, use `this.selectt()` to read the property of the state.

Looping all state changes through one state is a typical pattern for NgRx.